src/runtime/runtime.js

```diff
@@ -717,7 +717,17 @@ export class ConfiguredRuntime {
   getEntitlements(encryptedDocumentKey) {
     return this.entitlementsManager_
       .getEntitlements(encryptedDocumentKey)
-      .then(entitlements => entitlements.clone());
+      .then(entitlements => {
+        // Auto update internal things tracking the user's current SKU.
+        let sku = null;
+        if (entitlements) {
+          try {
+            sku = entitlements.getSingleEntitlement().getSku();
```

Comment:
Doesn't getSingleEntitlement() return the sku? I.e. should be "sku = entitlements.getSingleEntitlement();"?

Reply:
I've updated it to return the full entitlement instead

Reply:
I've managed to avoid needing this funciton.